public/index.html

```diff
@@ -0,0 +1,370 @@
+            <p class="column-right">Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industry standard dummy text since the 1500's, when an unknown printer took a galley of type and scrambled it to make a type specimen book</p>
+        </div>
+        <ul class="flex-container">
+            <li class="flex-item-1">
+                <h2>UI &#38; UX Design</h2>
+                <img src="./imgs/brush.gif" alt="brush logo">
+                <p>There are many variation of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+            </li>
+            <li class="flex-item-1">
+                <h2>LABORATORY</h2>
+                <img src="./imgs/erhlenmeyer.gif" alt="flask logo">
+                <p>There are many variation of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour</p>
+            </li>
+            <li class="flex-item-1">
+                <h2>DEVELOPMENT</h2>
+                <img src="./imgs/gear.gif" alt="gear logo">
+                <p>There are many variation of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+            </li>
+        </ul>
+        <h2 class="subheader">PRICING TABLES</h2>
+        <section class="table-section flex-container">
+            <table class="flex-item-3">
+                <thead>
+                    <tr class="table-head">
+                        <th>BASIC</th>
+                    </tr>
+                    <tbody>
+                        <tr>
+                            <td class="table-price">$5.99/MONTH</td>
+                        </tr>
+                        <tr>
+                            <td>3 Projects</td>
+                        </tr>
+                        <tr>
+                            <td>2 Users</td>
+                        </tr>
+                        <tr>
+                            <td>1 Gb user</td>
+                        </tr>
+                        <tr>
+                            <td>1 Calendar </td>
+                        </tr>
+                        <tr class="table-foot">
+                            <td>
+                                <button type="button" class="button-signup">SIGN UP</button>
+                            </td>
+                        </tr>
+                    </tbody>
+            </table>
+            <table class="flex-item-3">
+                <thead>
+                    <tr class="table-head">
+                        <th>STANDART</th>
+                    </tr>
+                </thead>
+                <tbody>
```

Comment:
Again, excessive table structuring. `thead` and `tbody` aren't necessary. But thank you for closing the `thead` this time.